src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectSystems/VsMSBuildProjectSystem.cs

```diff
@@ -285,26 +285,35 @@ public virtual void AddImport(string targetFullPath, ImportLocation location)
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
                 var relativeTargetPath = PathUtility.GetRelativePath(PathUtility.EnsureTrailingSlash(ProjectFullPath), targetFullPath);
-                AddImportStatement(relativeTargetPath, location);
-                await SaveProjectAsync();
```

Comment:
why are we not saving the project anymore?

Reply:
that will be taken care by ProjectHelper api itself.